mtp_noms_ops/apps/mtp_auth/forms.py

```diff
@@ -0,0 +1,52 @@
+import logging
+
+from django import forms
+from django.conf import settings
+from django.utils.translation import gettext as _
```

Comment:
be careful about importing `gettext` vs `gettext_lazy`. it's customary in django to import the lazy version as `_`, and in fact that's the one you want. not using the lazy version means that class-level attributes would be locked to the _first_ language that loads

Reply:
I see what you mean about the locking of the language 🤔 

Reason why I used the non-lazy version is because I'm using this here to translate a string that goes in the session (`Family Services` as for subsequent requests the message will be slightly different)...it turns out that you can't serialise the output of the lazy version 😬 

I thought that was simple and good enough but maybe we just use a simple constant in the session and make the translation in the view, I'll change 👍

Reply:
As discussed, removed and simplified this thanks to the fact success message will be always the same.